src/routes/api/commentRoute.js

```diff
@@ -0,0 +1,112 @@
+import Exists from '../../middlewares/Exists';
+import isUserVerified from '../../middlewares/isUserVerified';
+import AuthenticateToken from '../../helpers/AuthenticateToken';
+import commentAccess from '../../middlewares/commentAccess';
+import CommentController from '../../controllers/CommentController';
+
+const commentRoute = express.Router();
+
+/**
+* @swagger
+*
+* /trips/{tripRequestId}/comment:
+*   post:
+*     summary: User or manager can post a comment
+*     description: user/manager post comment
+*     tags:
+*       - Trip
+*     parameters:
+*      - name: token
+*        in: header
+*        required: true
+*        description: user token
+*        schema:
+*          type: string
+*        example: xxxxxxxxx.xxxxxxxxx.xxxxxxxxxx
+*        minimum: 1
+*      - name: tripRequestId
+*        in: path
+*        required: true
+*        description: Trip request id
+*        schema:
+*          type: integer
+*        example: 1
+*        minimum: 1
+*     requestBody:
+*       content:
+*         application/json:
+*           schema:
+*             type: object
+*             properties:
+*               comment:
+*                 type: string
+*     responses:
+*       201:
+*         description: Comment posted successfully
+*       400:
+*         description: Unable to post comment
+*       401:
+*         description: Unauthorized
+*       403:
+*         description: Only requesters and managers can post comments
+*       500:
+*         description: Internal server error
+*/
+/**
+* @swagger
+*
+* /trips/{tripRequestId}/comments:
+*   get:
+*     summary: User or manager can get a comment
+*     description: user/manager get comment
+*     tags:
+*       - Trip
+*     parameters:
+*      - name: token
+*        in: header
+*        required: true
+*        description: user token
+*        schema:
+*          type: string
+*        example: xxxxxxxxx.xxxxxxxxx.xxxxxxxxxx
+*        minimum: 1
+*      - name: tripRequestId
+*        in: path
+*        required: true
+*        description: Trip request id
+*        schema:
+*          type: integer
+*        example: 1
+*        minimum: 1
+*     responses:
+*       200:
+*         description: Comments retrieved successfully
+*       401:
+*         description: Unauthorized
+*       403:
+*         description: Only requesters and managers can get comments
+*       500:
+*         description: Internal server error
+*/
+commentRoute.post('/:tripRequestId/comment',
+  AuthenticateToken.verifyToken,
+  isUserVerified,
+  Validate.commentPostRules(),
+  checkInputDataError,
+  Exists.isTripRequestExist,
+  commentAccess,
+  CommentController.createComment);
+
+commentRoute.get('/:tripRequestId/comments',
+  AuthenticateToken.verifyToken,
+  isUserVerified,
+  Validate.getCommentsRules(),
+  checkInputDataError,
+  Exists.isTripRequestExist,
+  commentAccess,
+  CommentController.getComments);
+
+export default commentRoute;
```

Comment:
I saw in the other routes are arranged in a in a manner where the JSdoc follows its own route, why do you prefer this format?